src/scripts/clipperUI/componentBase.ts

```diff
@@ -158,6 +158,12 @@ export abstract class ComponentBase<TState, TProps> {
 					element.style.outlineStyle = "";
 				}
 			},
+			onkeydown: (e: KeyboardEvent) => {
+				if (e.which === Constants.KeyCodes.space) {
```

Comment:
If you press space in a text box, would this prevent the space character from being inputted?

Reply:
That still works fine- when it is in the textbox it still allows spaces and enter for paragraphs, this just directs you into that box

Reply:
Confirmed working